### DISCUSSION TITLE
Error en IntelliJ al importar un proyecto "Project JDK is not defined"

### DISCUSSION THREAD

**Discussion by LeoCubelli** [dds-jv/foro#186] [Consultas de herramientas] [answered]
Buenas, cloné el repo TPI Macowins del classroom y al importarlo en IntelliJ me tira el error "Project JDK is not defined", no encuentro la forma de definir que JDK usar, o si eso se hace en el archivo pom. Adjunto una imagen con el error y la consola, que según entiendo tengo bien instalado Maven y la JDK. 
![115322882-60067b00-a15d-11eb-9165-ff0f7ba66417](https://user-images.githubusercontent.com/62668099/115330473-7f57d500-a16a-11eb-90fc-1965e1f9981a.png)

**Comment by flbulgarelli** [comment, marked as answer]
¡Hola!

En principio el error es un problema de configuración de IntelliJ y no de maven ni del proyecto, que puede deberse a varios factores: por lo que entiendo no se trata de que el pom no esté definiendo la versión de Java (si te fijás, está especificada en el pom), sino que IntelliJ no está encontrando la instalación. Te dejo acá una guía oficial para hacerlo: https://www.jetbrains.com/help/idea/sdk.html#set-up-jdk

¡Saludos!

**Reply by flbulgarelli** [reply]
PD: [acá](https://qastack.mx/programming/18987228/how-do-i-change-the-intellij-idea-default-jdk) hay una versión del tutorial en español

**Reply by LeoCubelli** [reply]
Perfecto, creo que ahí funcionó (no me aparece más la línea azul)
![aaa](https://user-images.githubusercontent.com/62668099/115594941-b6360400-a2ac-11eb-9880-fa7eee7244d9.png)
Supongo que lo que me subraya en rojo es por no tener los métodos definidos
